queens-psyc-clinic/app
# FE Dynamic Modal Component

### **Modal Component**
Modal component complete! I also added DropDown, InputField, and RangeSlider components so it is easier to assemble each modal for every possible action (e.g. delete, confirmations, etc). 

### **Testing**
1. navigate to branch: `git checkout modal`
2. `cd` to the `public` folder
3. run `npm install`
4. run `npm start`
5. ensure the "+ Add" button opens the modal and all text fields are correct and buttons are working for you:)
<br>

When open the modal should look like this:

![Screenshot 2024-02-25 at 10 28 13 PM](https://github.com/queens-psyc-clinic/app/assets/74020381/f8554e11-8822-47d5-a8a3-727cbb4b539c)

For reference checkout issue: #10
